.circleci/config.yml

```diff
@@ -319,8 +316,7 @@ workflows:
   version: 2.1
   ci:
     jobs:
-      - lint-changes:
-          args: "--new-from-rev origin/next"
```

Comment:
I'd keep this, just base it on master

Reply:
Why? It just encourages people to ignore lint errors.